nullaway/build.gradle

```diff
@@ -48,7 +49,6 @@ dependencies {
     testImplementation deps.test.cfCompatQual
     testImplementation deps.build.jspecify
     testImplementation project(":test-java-lib")
-    testImplementation deps.test.inferAnnotations
```

Comment:
Should we just remove `inferAnnotations` entirely from `dependencies.gradle`?  Or is it still used somewhere?